cmd/quicksilverd/bulk_airdrop.go

```diff
@@ -241,8 +243,9 @@ func BulkGenesisAirdropCmd(defaultNodeHome string) *cobra.Command {
 					bankGenState.Balances = append(bankGenState.Balances, balances)
 					bankGenState.Supply = bankGenState.Supply.Add(balances.Coins...)
 				}
-			}
+				continue OUTER
 
+			}
```

Comment:
Is this needed?

I don't think this label jump is doing anything at all.  The behavior is same with or without it right?